pycobertura/reporters.py

```diff
@@ -204,43 +250,28 @@ def get_report_lines(self):
             )
         ]
 
-        filenames_of_files_with_changes = [
-            self.differ.files()[i] for i in indexes_of_files_with_changes
-        ]
-
         lines = {
-            "Filename": filenames_of_files_with_changes,
-            "Stmts": [
-                self.format_total_statements(diff_total_stmts[i])
-                for i in indexes_of_files_with_changes
-            ],
-            "Miss": [
-                self.format_total_misses(diff_total_miss[i])
-                for i in indexes_of_files_with_changes
-            ],
-            "Cover": [
-                self.format_line_rate(diff_total_cover[i])
-                for i in indexes_of_files_with_changes
-            ],
+            "Filename": [self.differ.files()[i] for i in indexes_of_files_with_changes]
+            + ["TOTAL"]
         }
 
-        lines["Filename"].append("TOTAL")
-        lines["Stmts"] += [
-            self.format_total_statements(self.differ.diff_total_statements())
-        ]
-        lines["Miss"] += [self.format_total_misses(self.differ.diff_total_misses())]
-        lines["Cover"] += [self.format_line_rate(self.differ.diff_line_rate())]
+        for key in set(headers_hideable).difference(set(self.hide_columns)):
```

Comment:
Can we store the output of `difference()` in a variable in order to label it?

```suggestion
        headers_to_show = set(headers_hideable).difference(set(self.hide_columns))
        for header_name in headers_to_show:
```

Reply:
We should be consistent on the naming used, "columns" vs. "headers". Can we pick one and be consistent throughout the code?